consumer/go.mod

```diff
@@ -53,3 +53,5 @@ retract (
 replace go.opentelemetry.io/collector/config/configtelemetry => ../config/configtelemetry
 
 replace go.opentelemetry.io/collector/config/configretry => ../config/configretry
+
+replace go.opentelemetry.io/collector/config/configqueue => ../config/configqueue
```

Comment:
How does this gets here? Seems that we have some sort of circular dependency via the `go.opentelemetry.io/collector` module.

Reply:
It comes from `obsreport/obsreporttest` which is not a separate module. We need to get rid of that package